src/backend/feed.js

```diff
@@ -0,0 +1,33 @@
+const storage = require('./utils/storage');
+
+/**
+ * Feed Object, upon being created all feeds will be active and have a feedId of 0 until saved into Redis
+ * @param url url of feed
+ * @param name author name of feed
+ * @param feedId unique id for feed
+ */
+class Feed {
+  constructor(url, name) {
+    this.url = url;
+    this.name = name;
+    this.inactive = false;
+    this.feedId = 0;
+  }
+
+  /**
+   * Saves current feed to appropriate set in Redis
+   */
+  async save() {
+    this.feedId = await storage.addFeed(this);
+  }
+
+  /**
+   * Marks the feed as inactive
+   */
+  inactive() {
+    this.inactive = true;
```

Comment:
Instead of storing state for this, put it in the appropriate set.

How we we re-activate a feed?

Reply:
on it, added an active() function that will do the opposite and adjusted the storage.js addFeed to handle it as well